apps/rush-lib/src/logic/PackageJsonUpdater.ts

```diff
@@ -316,14 +312,26 @@ export class PackageJsonUpdater {
       console.log();
       console.log(`Querying registry for all versions of "${packageName}"...`);
 
+      let commandArgs: Array<string> = ['view', packageName, 'versions', '--json'];
```

Comment:
Let's put this in an explicit `else` block, similar to the code starting on line 329